packages/common/src/dotlottie-player.ts

```diff
@@ -1502,10 +1518,14 @@ export class DotLottiePlayer {
     // If loop = number, and animation has reached the end, call stop to go to frame 0
     if (typeof this._loop === 'number') this.stop();
 
-    this._container?.dispatchEvent(new Event(PlayerEvents.Complete));
+    const lastFrame = this.direction === -1 ? 0 : this.totalFrames;
+
+    this._lottie?.goToAndStop(lastFrame, true);
```

Comment:
@afsalz Can we use 

this.goToAndStop() rather than going through the Lottie method directly.

This would help us keeping track of whats controlling the player